src/main/ngapp/src/app/main/main.component.ts

```diff
@@ -208,23 +239,33 @@ export class MainComponent implements OnInit {
       const messagePromise = this.fetchMessages(chatroomId);
       messagePromise
         .then((messages) => {
+          this.chatSectionLoading = true;
           console.log(messages);
         })
         .catch((err) => {
+          this.chatSectionLoading = true;
```

Comment:
why would the chatRoomSection be loading when there is an error?

Reply:
Should be false. Updated